docs/configuration/configuration.md

```diff
@@ -206,6 +206,10 @@ digitalocean_sd_configs:
 dockerswarm_sd_configs:
   [ - <dockerswarm_sd_config> ... ]
 
+# List of Docker service discovery configurations.
+docker_sd_configs:
```

Comment:
This needs to be also later on, in the alertmanager_config section.

Reply:
I've added it to the other section as well. It first looked like other SDs are missing from the alertmanager_config section as well, but it turns out that the list is just sorted a bit differently. Is this something that you want fixed (possibly in a separate PR?)

Reply:
Yes let's have it in a split PR after this one is merged :)